beacon-chain/powchain/check_transition_config.go

```diff
@@ -0,0 +1,62 @@
+package powchain
+
+import (
+	"context"
+	"errors"
+	"math/big"
+	"time"
+
+	v1 "github.com/prysmaticlabs/prysm/beacon-chain/powchain/engine-api-client/v1"
+	"github.com/prysmaticlabs/prysm/config/params"
+	pb "github.com/prysmaticlabs/prysm/proto/engine/v1"
+)
+
+var (
+	checkTransitionPollingInterval = time.Minute
+	configMismatchLog              = "Configuration mismatch between your execution client and Prysm. Node is shutting down as it" +
+		" will not be able to complete the proof-of-stake transition"
```

Comment:
```suggestion
	configMismatchLog              = "Configuration mismatch between your execution client and Prysm. Node is shutting down as it " +
		"will not be able to complete the proof-of-stake transition"
```

Minor nitpick, a new section of a string break should not start with a space.